src/controllers/Auth.ts

```diff
@@ -389,11 +400,21 @@ export class AuthController extends BaseController {
       strategy,
       expiresIn,
       body: credentials,
-      action: 'login'
+      action: 'login',
+      cookieAuth: this.kuzzle.cookieAuthentication
     };
 
     return this.query(request, { queuable: false, verb: 'POST' })
       .then(response => {
+        if (this.kuzzle.cookieAuthentication) {
+          if (response.result.jwt) {
+            throw new Error('Kuzzle support for cookie authentication is disabled or not supported');
+          }
```

Comment:
Yes you're right, it seems that if cookieAuth is set and the CORS headers are unconfigured, Kuzzle should throw that this is unsupported.
I will make a PR to fix this behaviour, IDK if this needs to be a hotfix or not since this is not critical.

Reply:
I don't think that a hotfix is in order either.